cmd/sovereignnode/chainSimulator/tests/bridge/bridge_test.go

```diff
@@ -0,0 +1,221 @@
+package bridge
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	chainSimulatorIntegrationTests "github.com/multiversx/mx-chain-go/integrationTests/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components/api"
+	sovereignChainSimulator "github.com/multiversx/mx-chain-go/sovereignnode/chainSimulator"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	coreAPI "github.com/multiversx/mx-chain-core-go/data/api"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	defaultPathToInitialConfig = "../../../../node/config/"
+	sovereignConfigPath        = "../../../config/"
+	esdtSafeWasmPath           = "../testdata/esdt-safe.wasm"
+	feeMarketWasmPath          = "../testdata/fee-market.wasm"
+	multiSigWasmPath           = "../testdata/multisigverifier.wasm"
+)
+
+var oneEgld = big.NewInt(1000000000000000000)
+var initialMinting = big.NewInt(0).Mul(oneEgld, big.NewInt(100))
+var issueCost = big.NewInt(5000000000000000000)
+
+func TestBridge_DeployOnMainChain_IssueAndDeposit(t *testing.T) {
```

Comment:
Why are we creating this tests for main chain simulator on this sovereign branch instead of targeting it towards rc branch? This test is also testing main chain behavior inside sovereign chain package